code/modules/mob/living/carbon/human/human.dm

```diff
@@ -2026,3 +2026,37 @@ mob/living/carbon/human/isincrit()
 		return list(/datum/ambience/beach)
 	else
 		return ..()
+
+//Start audible emote code
+
+var/list/male_screams =  list('sound/misc/malescream1.ogg', 'sound/misc/malescream2.ogg', 'sound/misc/malescream3.ogg', 'sound/misc/malescream4.ogg', 'sound/misc/malescream5.ogg', 'sound/misc/wilhelm.ogg', 'sound/misc/goofy.ogg')
+var/list/female_screams = list('sound/misc/femalescream1.ogg', 'sound/misc/femalescream2.ogg', 'sound/misc/femalescream3.ogg', 'sound/misc/femalescream4.ogg', 'sound/misc/femalescream5.ogg')
+
+var/list/vox_shrieks = list('sound/misc/shriek1.ogg')
+
+var/list/insectoid_chittering = list('sound/misc/hiss1.ogg', 'sound/misc/hiss2.ogg', 'sound/misc/hiss3.ogg')
+
+var/list/male_coughs = list('sound/misc/cough/cough_m1.ogg', 'sound/misc/cough/cough_m2.ogg', 'sound/misc/cough/cough_m3.ogg', 'sound/misc/cough/cough_m4.ogg')
+var/list/female_coughs = list('sound/misc/cough/cough_f1.ogg', 'sound/misc/cough/cough_f2.ogg', 'sound/misc/cough/cough_f3.ogg', 'sound/misc/cough/cough_f4.ogg')
+
+//Run a bunch of arbitrary checks based on code in code/modules/mob/living/carbon/emote.dm
+/mob/living/carbon/human/audible_emote_sound(emote)
+	switch(emote)
```

Comment:
Isn't avoiding massive switch()es the whole point of having emote datums? Why isn't all this code inside emote datums?

Reply:
1. The PR was already starting to get bloated, so I decided to relegate further clean-up to a future PR. Ideally this sort of check would happen at the emote level running certain conditions, such as automatically redirecting a scream to a shriek/chittering emote based on the race, which again I hope I will get to work on a future PR
2. It was moved there because it's a mob proc to run a bunch of conditions and then select the proper audible emote, it feels lackluster because only humanoids can use emotes currently

Reply:
Nope, this won't fly. It needs to be an emote-level proc, you can pass the mob as an argument if you need to use a different path depending on the gender/species, and let the emote datums themselves pick one. This switch needs to go.

Reply:
kay